src/view/PrescriptionView.vue

```diff
@@ -1,9 +1,140 @@
+                </button-component>
+
+                <teleport to="body">
+                    <transition name="modal">
+                        <div
+                            v-if="openModalPrescription"
+                            class="modal">
+                            <modal-component
+                                ref="modalPrescription"
+                                @close="openModalPrescription = false">
+                                <prescription-add-component />
+                            </modal-component>
+                        </div>
+                    </transition>
+                </teleport>
+
+                <button-component
+                    class="prescription__make_visit"
+                    @click="openModalImport = true">
+                    Importuj receptę
+                    <font-awesome-icon icon="fa-solid fa-plus" />
+                </button-component>
+
+                <teleport to="body">
+                    <transition name="modal">
+                        <div
+                            v-if="openModalImport"
+                            class="modal">
+                            <modal-component
+                                ref="modalImport"
+                                @close="openModalImport = false">
+                                <prescrition-import />
+                            </modal-component>
+                        </div>
+                    </transition>
+                </teleport>
+            </div>
+            <article class="prescription__listOf">
+                <prescription-group-component />
+            </article>
+
+            <article class="prescription__import">
+                <import-group-component />
+            </article>
+        </div>
+    </section>
 </template>
 
-<script setup></script>
+<script setup>
+    import ButtonComponent from '../components/ButtonComponent.vue';
+    import useUserStore from '../stores/userStore.js';
+    import ModalComponent from '../components/dashboard/ModalComponent.vue';
+    import PrescriptionGroupComponent from '../components/prescriptions/PrescriptionGroupComponent.vue';
+    import GreetingsComponent from '../components/GreetingsComponent.vue';
+    import PrescriptionAddComponent from '../components/prescriptions/PrescriptionAddComponent.vue';
+    import PrescritionImport from '../components/prescriptions/PrescritionImport.vue';
+    import ImportGroupComponent from '../components/prescriptions/ImportGroupComponent.vue';
+
+    import { onMounted, ref } from 'vue';
+    import { storeToRefs } from 'pinia';
+    import { onClickOutside } from '@vueuse/core';
+
+    const userStore = useUserStore();
+    const { getUserInfo } = storeToRefs(userStore);
+    const openModalPrescription = ref(false);
+    const openModalImport = ref(false);
+    const modalPrescription = ref(null);
+    const modalImport = ref(null);
+
+    onClickOutside(
```

Comment:
Please standarize these two calling function onlickoutside in one line or both divided into few lines